core/templates/dev/head/pages/terms/terms.html

```diff
@@ -91,9 +91,10 @@
         is then immediately licensed by us under the Creative Commons
         CC-BY-SA 4.0 license with a waiver of the attribution (BY) requirement,
         which means that all of the content on Oppia is irrevocably licensed to
-        the general public and can be freely used, reused, and shared. You can
-        read more about the CC-BY-SA 4.0 license
-        <a href="https://creativecommons.org/licenses/by-sa/4.0/legalcode">here</a>.
+        the general public and can be freely used, reused, and shared. Read
```

Comment:
Ah -- please could we hold off on changing this file for now, and defer modifications to a future PR? (I need to check what we can do re modifying the actual content of this page.)